tests/integration_test/backend_auth_using_imds_test.go

```diff
@@ -61,7 +62,8 @@ func TestBackendAuthWithImdsIdToken(t *testing.T) {
 			desc:     "Add Bearer token for APPEND_PATH_TO_ADDRESS backend that requires JWT token",
 			method:   "GET",
 			path:     "/bearertoken/append?key=api-key",
-			wantResp: `{"Authorization": "Bearer ya29.append", "RequestURI": "/bearertoken/append?key=api-key"}`,
+			headers:  map[string]string{"Authorization": "Bearer origin-token"},
+			wantResp: `{"Authorization": "Bearer ya29.append", "RequestURI": "/bearertoken/append?key=api-key", "X-Forwarded-Authorization":"Bearer origin-token"}`,
```

Comment:
I suggest you keep this test case as-is and add a new test case. Change the description to something like: "Downstream authorization header is copied when upstream requires JWT token".

Also consider adding a test to `backend_auth_using_iam_test.go` for completeness

https://github.com/GoogleCloudPlatform/esp-v2/blob/fc0d4fbb331e6c8e1bc452bc26a0d318c409d7fd/tests/integration_test/backend_auth_using_iam_test.go#L61

Reply:
Done, add two cases in backend_auth_disable_auth_test.go